db/migration/000002_cli_user.up.sql

```diff
@@ -0,0 +1,25 @@
+
+ALTER TABLE users ADD COLUMN created_by UUID;
+-- SQL Definition for environments
+
+CREATE TABLE environments (
+    Id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    cli_user UUID NOT NULL REFERENCES users(id),
+    name VARCHAR(255) NOT NULL,
+    createdAt TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    updatedAt TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    created_by UUID NOT NULL REFERENCES users(id)
+);
+-- SQL Definition for environment fields
+
+CREATE TABLE environment_fields (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    field_name VARCHAR(255) NOT NULL,
+    field_value TEXT NOT NULL,
+    parent_field_id UUID NOT NULL REFERENCES fields(id),
+    env_id UUID NOT NULL REFERENCES environments(Id),
+    cli_user UUID NOT NULL REFERENCES users(id),
```

Comment:
why do we need cli_user here if one env is tied to a single user

Reply:
cli_user can be removed.